docs/guides/migrating-from-camunda-7/conceptual-differences.md

```diff
@@ -46,7 +46,21 @@ To migrate existing Connectors, consider the following options:
 - Use the [REST protocol Connector](components/connectors/protocol/rest.md) to leverage an out-of-the-box Connector.
 - Create a small bridging layer via custom [job workers](/components/concepts/job-workers.md).
 
-### Process solutions using Spring Boot
+### Multi-tenancy
+
+:::note
+[Multi-tenancy](/self-managed/concepts/multi-tenancy.md) is currently in development for Camunda 8.
+:::
+
+There are several differences between how [multi-tenancy](/self-managed/concepts/multi-tenancy.md) works in Camunda 7 and Camunda 8:
+
+1. The one engine per tenant approach from Camunda 7 isn't possible with Camunda 8. Camunda 8 only provides multi-tenancy through a tenant identifier.
+2. In Camunda 7, users can deploy shared resources (processes, decisions, and forms) to all tenants. In Camunda 8, there are no shared resources. This will be added in the future.
+3. In Camunda 7, data is mapped to a `null` tenant identifier, meaning by default resources are shared. In Camunda 8, data is mapped to the `<default>` tenant identifier when multi-tenancy is disabled.
+4. Tenant checks in Camunda 7 can be disabled to perform admin/maintenance operations. This can't be done in Camunda 8, but an admin user can be authorized to all tenants, which would result in the same thing.
```

Comment:
:wrench: We can link to the appropriate C7 docs here: https://docs.camunda.org/manual/develop/user-guide/process-engine/multi-tenancy/#disable-the-transparent-access-restrictions

Reply:
Resolved in latest commit 👍